### PR TITLE
Spatial filter implementation - Backend

### PR DESCRIPTION
**Pull request by dimasciput**
- [x] Added a function to extract spatial data from the intersection of the context layer and the property
  - [x] Todo: add tests

### CODE REVIEW

File: django_project/property/signals.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@receiver(post_save, sender=Property)
+def handle_spatial_data(sender, instance, created, **options):
+    if created:
+        context_layers = ContextLayer.objects.all()
+
+        for context_layer in context_layers:
+            # Create spatial data instance
+            spatial_data_instance = SpatialDataModel(
+                property=instance,
+                context_layer=ContextLayer.objects.get(id=context_layer.id)
```
Comment (dimasciput) [inline django_project/property/signals.py:19]: Why do you query the context layer again here? Can you just use the context_layer.id?

**Comment by dimasciput** [comment]
This is for ticket https://github.com/kartoza/sawps/issues/1155

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 43 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

**Pull request merged**